libdnf/module/module_sack.cpp

```diff
@@ -135,7 +140,6 @@ void ModuleSack::create_module_solvables() {
     // dnf_sack_set_considered_to_update(moduleSack);
 }
 
-
```

Comment:
Why is this removed?

Reply:
No idea, may be it is a consequence of code modification that was removed